frontend/views/components/LinkToCopy.vue

```diff
@@ -47,14 +47,25 @@ export default ({
     }
   },
   methods: {
+    // FIXME: probably needs to be defined centrally somewhere else
+    isMacSafari (userAgent) {
+      const safari = /Macintosh.*Safari/
+      const chrome = /Macintosh.*Chrome/
+      const edge = /Macintosh.*Edg/
+      return userAgent.match(safari) &&
+              !(userAgent.match(chrome) || userAgent.match(edge))
+    },
     copyToClipboard () {
       // if the user is using the device that supports web share API, use it and then skip the other logics below.
       if (navigator.share) {
         navigator.share({
           title: this.L('Your invite'),
           url: this.link
-        })
-        return
+        }).catch((error) => console.log(error))
```

Comment:
Errors should be logged with `console.error` and should always indicate which part of the code caused them, so something like this would be better:

```js
        }).catch((error) => console.error('navigator.share failed with:', error))
```

Reply:
good point, changed